### ISSUE TITLE
Fails on Java 8

### ISSUE THREAD

**Issue by jamespamplin** [guardian/composer-restorer#43]
openjdk-7 / oracle java 7 works fine.

openjdk-8 / oracle java 8 doesn't retrieve versions from S3 properly.

**Comment by theefer** [comment]
Looks like the CF script installs java 7. Doesn't it make sense to use the same version of java locally as in production (as the recent issues taught us), or are you trying to upgrade the deployed version to Java 8?

**Comment by jamespamplin** [comment]
@theefer yeah everyone should be on Java 7 really. But wanted to note it here for future ref.

**Comment by theefer** [comment]
Fair enough.
